src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,46 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  constructor(health, name, hidden) {
+    this.name = name;
+    this.hidden = hidden || false;
```

Comment:
Each Animal can't have this field
```suggestion

```

Reply:
isn't fixed. Please remove the line previous mentor mentioned